rust-semantics/representation.md

```diff
@@ -98,7 +98,7 @@ module RUST-REPRESENTATION
     syntax MapOrError ::= Map | SemanticsError
 
     syntax Expression ::= Ptr
-    syntax ExpressionOrError ::= Expression | SemanticsError
+    syntax ExpressionOrError ::= v(Expression) | e(SemanticsError)
```

Comment:
This change is needed because of a K crash that does not make sense, and looks like this:
```
(NoSuchElementException: key not found: inj{SortExpressionOrError{}, SortKItem{}})
```

These seem to be triggered by random changes in the code, not related in any way to the ExpressionOrError sort. Anyway, using explicit constructors seems to make the error go away (it may also be that this is another random code change that makes the error go away for no good reason).